Packages/com.unity.inputsystem/InputSystem/Actions/Composites/Vector3Composite.cs

```diff
@@ -46,7 +46,7 @@ public class Vector3Composite : InputBindingComposite<Vector3>
         /// </remarks>
         // ReSharper disable once MemberCanBePrivate.Global
         // ReSharper disable once FieldCanBeMadeReadOnly.Global
-        [InputControl(layout = "Axis")] public int up;
+        [InputControl(layout = "Axis")] public int up = 0;
```

Comment:
Not sure these need explicit initialisation since default(int) is known to be zero in C#?

Reply:
That makes sense, in that case I'll change the Vector2Composite to remove the explicit initialisation - just noticed some discrepancies in consistency and wanted to clean them up!

Reply:
Resolved in f15b4feb5bbb9acf439cb77e17d2396ef6f69e04.

Reply:
Ah, you decide what is best, consistency wins, not a big deal either way.

Reply:
This change broke the test API_MinorVersionsHaveNoBreakingChanges on my branch.
It took me a while to trace it here because CI on develop and on this branch both seemed to have passed... 😸